### PR TITLE
Add mailtrap-subject-copier extension

### PR DESCRIPTION
**Pull request by samjwillis97**
## Description

This extensions interacts with Mailtrap's API to allow for quickly copying information from the subject line of emails inside of Mailtrap inboxes

## Screencast

https://github.com/raycast/extensions/assets/37866085/0182f72f-28ea-474a-9377-6972d93a027a

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/mailtrap-subject-copier/src/views/list-inboxs.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  return (
+    <List isLoading={isLoading}>
+      {(data || []).map((inbox) => (
+        <List.Item
+          key={inbox.id}
+          title={inbox.name}
+          accessories={[{ icon: Icon.Eye, text: `${inbox.emails_unread_count} / ${inbox.emails_count}` }]}
+          actions={
+            <ActionPanel>
+              <Action.Push title="Open Inbox" target={<ListEmailsView inboxId={inbox.id} />} />
```
Comment (pernielsentikaer) [inline extensions/mailtrap-subject-copier/src/views/list-inboxs.tsx:29]: Missing an icon 🙂

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

Is there a reason why this only focuses on the subject? Wouldn't it be nice if it were a MailTrap extension instead?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by samjwillis97** [comment]
Thanks for the review 😊 I'll get those changes sorted very soon.

Also, I do agree this would ideally be a more general Mailtrap extension and that was my intentions when starting this, though as I was building I realised the only use I had was copying the subject and wasn't entirely sure what other commands would be useful to people 😅

I just really wanted to get something published and I'm more than happy to take on suggestions to expand this further!

**Comment by pernielsentikaer** [comment]
We can indeed make an initial version that can be expanded later on, but I would think we should add the correct name from start and maybe in the README describe what would be added later on (or missing)

**Comment by samjwillis97** [comment]
Sounds good, so should I go with the name Mailtrap then? and explain reasoning etc. in the README?

**Comment by pernielsentikaer** [comment]
I would do that, it's okay that features are missing 🙂

**Comment by samjwillis97** [comment]
Thanks for fixing up that API key field, is the updated README alright?

**Pull request merged**